src/table/table.scss

```diff
@@ -16,75 +15,205 @@
+        }
+      }
+
+      &:hover > .iui-resizer-bar {
+        width: $iui-xs;
+        @include themed {
+          background-color: t(iui-color-foreground-primary);
+        }
+      }
+    }
+
+    &:hover,
+    &:focus,
+    &:focus-within {
+      @include themed {
+        background-color: t(iui-color-background-4);
+      }
 
+      .iui-sort,
+      .iui-borderless {
+        visibility: visible;
+      }
+    }
+
+    &:hover {
+      > .iui-resizer {
+        display: block;
+      }
+    }
+  }
+
+  // Sort icon
+  .iui-sort {
+    visibility: hidden;
+    margin-left: auto;
+    width: $iui-icons-default;
+    height: $iui-icons-default;
     @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
+      fill: t(iui-icons-color-actionable);
     }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
+  // Sorted column
+  .iui-sorted {
+    @include themed {
+      background-color: t(iui-color-background-4);
+    }
+
+    .iui-sort {
+      visibility: visible;
+        
+      &,
+      > * {
+        @include themed {
+          fill: t(iui-icons-color-actionable);
+        }
+      }
+    }
+  }
+
+  // Double header
+  .iui-row + .iui-row {
+    @include themed {
+      border-top: 1px solid t(iui-color-background-5);
     }
+  }
+}
+
+@mixin iui-table-body {
+  width: 100%;
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  
+  .iui-row {
+    display: flex;
+    flex-grow: 1;
+    width: 100%;
+    margin-top: -1px;
+    border: solid 1px transparent;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
+    // Last row doesn't get bottom border. Up for debate.
+    &:not(:last-child) {
       @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
+        border-bottom: solid 1px t(iui-color-background-4);
+      }
+    }
+
+    > .iui-slot > .iui-more-options {
+      visibility: hidden;
+    }
+
+    &:hover:not(.iui-disabled) {
+      &:not(.iui-selected):not(.iui-expanded-content) {
+        @include themed {
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          );
+        }
       }
 
-      > .iui-tables-slot > .iui-tables-menu {
+      > .iui-slot > .iui-more-options {
         visibility: visible;
       }
     }
 
-    > .iui-tables-cell {
+    &:focus-within > .iui-slot > .iui-more-options {
+      visibility: visible;
+    }
+
+    > .iui-cell {
+      position: relative;
       align-items: center;
       display: flex;
       flex-grow: 1;
-      min-width: 145px;
-      padding: $iui-table-density;
+      min-width: $iui-l * 6;
+      min-height: $iui-baseline * 5;
+      padding: 0 $iui-m;
       overflow: hidden;
       list-style: none;
-      flex-basis: 145px;
+      flex-basis: $iui-l * 6;
 
-      > svg {
+      > .iui-icon {
```

Comment:
So is this only for the start icon???
Gonna be interesting in React...

Reply:
I believe the `margin-right` is intended for the filetype icon within the Name column.  This could probably be moved out of `.iui-cell` and instead be placed within `.iui-main-column`.  The width & height could stay because we'd likely recommend a 16x16 icon in the table, but I could see the argument for removing that and making it user defined.

I don't believe this CSS Is for the expandable row only because the expand/collapse icons are wrapped within a borderless button.